.hgtags

```diff
@@ -0,0 +1 @@
+fee87bfa9db131c0a02ba5d0bdfb910bc07994f3 0.8.0
```

Comment:
Intentional?

Reply:
Btw, we also need to make another release candidate to upload to Zenodo.

Reply:
I removed this file. I never tried to great a git tag with hg-git, and actually I don't know how to do that. Anyway, for now I won't do it from hg...

Ok for the release candidate.